libtcmu.c

```diff
@@ -924,6 +928,27 @@ int tcmu_dev_get_fd(struct tcmu_device *dev)
 	return dev->fd;
 }
 
+/**
+ * tcmu_dev_get_memory_info - retrieve information about tcmu's shared memory block.
+ * If the memory is mapped to another address (e.g., in another process) then the
+ * information in struct iovec can be converted using these values.  Returns
+ * the filename of the mmap() that fd uses, or NULL if information is not
+ * available.
+ * @dev: tcmu device
+ * @base: receives the base address of the shared memory region
+ * @len: receives the size of the shared memory region
+ * @offset: receives the offset within fd to the shared memory region (conventionally 0)
+ */
+const char *
+tcmu_dev_get_memory_info(struct tcmu_device *dev, void **base, size_t *len, off_t *offset)
+{
+	if (!dev->mmap_name) return NULL;
+	if (base) *base = dev->map;
+	if (len) *len = dev->map_len;
+	if (offset) *offset = FDOFFSET;
```

Comment:
Just follow the coding style in the rest of the code and put what comes after the test on a new line.